### PR TITLE
Create nicer console titles #152

### LINKED ISSUE #152: Nicer console titles

### PR DESCRIPTION
**Pull request by seanfarmar**
closes #152

### CODE REVIEW

**Comment by seanfarmar** [comment]
Fixed the reference issue

**Comment by mauroservienti** [comment]
I'm on the fence. Since we usually run HashBus from a singl VM using Consoles and not Windows Services changing the console title is comfortable because it allows to easily identify each console instance.  It other than this I don't think there is much value.

File: src/HashBus.Viewer.TopTweetersRetweeters/Program.cs [hunk 1]
```diff
@@ -1,11 +1,14 @@
 ﻿namespace HashBus.Viewer.TopTweetersRetweeters
 {
+    using System;
     using System.Configuration;
 
     class Program
     {
         static void Main()
         {
+            Console.Title = "ashBus.Viewer.TopTweetersRetweeters";
```
Comment (adamralph) [inline src/HashBus.Viewer.TopTweetersRetweeters/Program.cs]: Typo here.

File: HashBus.sln.DotSettings [hunk 2]
```diff
@@ -586,6 +586,9 @@ II.2.12 &lt;HandlesEvent /&gt;&#xD;
...
 	<s:Boolean x:Key="/Default/Environment/SettingsMigration/IsMigratorApplied/=JetBrains_002EReSharper_002EPsi_002ECSharp_002ECodeStyle_002ESettingsUpgrade_002EMigrateThisQualifierSettings/@EntryIndexedValue">True</s:Boolean>
 	<s:Boolean x:Key="/Default/Environment/SettingsMigration/IsMigratorApplied/=JetBrains_002EReSharper_002EPsi_002EJavaScript_002ECodeStyle_002ESettingsUpgrade_002EJsCodeFormatterSettingsUpgrader/@EntryIndexedValue">True</s:Boolean>
+	<s:Boolean x:Key="/Default/Environment/SettingsMigration/IsMigratorApplied/=JetBrains_002EReSharper_002EPsi_002EJavaScript_002ECodeStyle_002ESettingsUpgrade_002EJsParsFormattingSettingsUpgrader/@EntryIndexedValue">True</s:Boolean>
+	<s:Boolean x:Key="/Default/Environment/SettingsMigration/IsMigratorApplied/=JetBrains_002EReSharper_002EPsi_002EJavaScript_002ECodeStyle_002ESettingsUpgrade_002EJsWrapperSettingsUpgrader/@EntryIndexedValue">True</s:Boolean>
+	<s:Boolean x:Key="/Default/Environment/SettingsMigration/IsMigratorApplied/=JetBrains_002EReSharper_002EPsi_002EXml_002ECodeStyle_002EFormatSettingsUpgrade_002EXmlMoveToCommonFormatterSettingsUpgrade/@EntryIndexedValue">True</s:Boolean>
```
Comment (adamralph) [inline HashBus.sln.DotSettings]: What is the purpose of the changes to this file? Are they required for this PR?

Comment (seanfarmar) [inline HashBus.sln.DotSettings]: I'm not sure...

Comment (mikeminutillo) [inline HashBus.sln.DotSettings]: https://resharper-support.jetbrains.com/hc/en-us/community/posts/206653075--sln-DotSettings-file-keeps-being-changed

Comment (adamralph) [inline HashBus.sln.DotSettings]: Grrrr... ReSharper slowing things down again :trollface: 

@seanfarmar can you please revert these changes from this PR. We can update the ReSharper settings according the current version in https://github.com/Particular/RepoStandards in a separate PR.

Comment (seanfarmar) [inline HashBus.sln.DotSettings]: Sure

**Comment by seanfarmar** [comment]
@adamralph reverted the resharper added lines in the .DotSettings file https://github.com/Particular/HashBus/pull/153/commits/22a2cdcc383fea10dc628d82b4030a47e0238c1d

**Comment by seanfarmar** [comment]
@adamralph 
As for the console title and endpoint name, I'd be happy to make them consistent and use a constant in the app.cs

As for where do we set it, I think it makes it simpler to have it in App.cs file

Shall i go ahead and make the change to the have console title and endpoint name the same?

File: HashBus.sln.DotSettings [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
```
Comment (adamralph) [inline HashBus.sln.DotSettings]: This is a change in encoding, from ASCII to UTF-8 BOM. You should be able to revert it with:

```bash
git checkout 1290ddb7c19eb122a0405d5f354658f8b74599e1 .\HashBus.sln.DotSettings
```

and then commit the change (no need to `git add`).
